doc/Command-Reference.md

```diff
@@ -9946,8 +9947,55 @@ In SONiC, there usually exists a set of tables related/relevant to a particular
 	    }
 	}	
   ```
+
+### Event Driven Techsupport Invocation
+
+This feature/capability makes the techsupport invocation event-driven based on core dump generation. This feature is only applicable for the processes running in the containers. More detailed explanation can be found in the HLD https://github.com/Azure/SONiC/blob/master/doc/auto_techsupport_and_coredump_mgmt.md
+
+- Usage:
+  ```
+  1) Configuration:
+  config auto-techsupport global state <enabled/disabled>
```

Comment:
a question in general, should we have more info and examples for the config part? I believe user should not go to the HLD for CLI commands only if he would like to understand the internals. So my questions is what is the minimum info we can provide to users for the CLI documentation to avoid going to an HLD.

Reply:
For the users to tune the configuration options, they have to understand them, and for that best place is to visit HLD. I believe any more description about those parameters here would be a duplicate information and not necessary. 

Let me know if you think otherwise